src/test/kotlin/model/ResultCalculatorTest.kt

```diff
@@ -0,0 +1,32 @@
+package model
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+
+class ResultCalculatorTest {
```

Comment:
👍

Reply:
How about adding new tests for `BlackJack`, since there are new logics?
Similarly, if you're planning new objects(e.g., `Earning`), new tests for them would be beneficial.